EventsExpress.Core/Services/AuthService.cs

```diff
@@ -234,16 +241,16 @@ private async Task<Account> ConfirmEmail(CacheDto cacheDto)
             return authLocal.Account;
         }
 
-        public Guid GetCurrUserId(ClaimsPrincipal userClaims)
+        public UserDto GetCurrentUser()
         {
-            Claim guidClaim = userClaims.FindFirst(ClaimTypes.Name);
+            Claim guidClaim = _httpContextAccessor.HttpContext.User.FindFirst(ClaimTypes.Name);
```

Comment:
Why did not we use ISecurityContext here and remove dependency on HttpContextAccessor?